go.sum

```diff
@@ -30,6 +30,8 @@ github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA
 github.com/cespare/xxhash/v2 v2.1.1/go.mod h1:VGX0DQ3Q6kWi7AoAeZDth3/j3BFtOZR5XLFGgcrjCOs=
 github.com/ciscoecosystem/aci-go-client/v2 v2.21.0 h1:7TXGLg18G/GUdc6eFG8MbnMDgs0Uw+tXcoLPQA6kYO0=
 github.com/ciscoecosystem/aci-go-client/v2 v2.21.0/go.mod h1:0uk5hiHANgXo+0h+Z8dFBriB7hHnI0PSGVUm7/20bt4=
```

Comment:
Remove the old v2.21.0 lines from go.sum

```suggestion
```

Reply:
Vendor files have been added